activemq-broker/src/main/java/org/apache/activemq/broker/region/policy/DeadLetterStrategy.java

```diff
@@ -60,6 +60,16 @@ public interface DeadLetterStrategy {
      */
     public void setProcessNonPersistent(boolean processNonPersistent);
 
+    /**
+     * @return the PreserveDeliveryMode
+     */
+    public boolean isPreserveDeliveryMode();
+
+    /**
+     * @param PreserveDeliveryMode the PreserveDeliveryMode to set
+     */
+    public void setPreserveDeliveryMode(boolean PreserveDeliveryMode);
```

Comment:
preserveDeliveryMode

Reply:
Good spot, not sure how I made this mistake.